### DISCUSSION TITLE
spinning={spinning} vs spinning && <Spin />

### DISCUSSION THREAD

**Discussion by James2516** [ant-design/ant-design#51711] [General]
### What problem does this feature solve?

Please clarify which approach is better: `<Spin spinning={spinning} />` or `{spinning && <Spin />}`?

### What does the proposed API look like?

N/A

**Comment by farmer00317558** [comment]
The behavior of Span is very strange.

- <Spin /> is a spinning Icon
- <Spin indicator={<LoadingOutlined />} /> change the icon but stop spinning. Only change icon will be more make scese.
- <Spin indicator={<LoadingOutlined />} spinning /> change the icon but also stop spinning. 
-  <Spin indicator={<LoadingOutlined spin />} /> change the icon and spinning. But I cant place a custom icon in the Spin.
